usd_qtpy/render_util/framing_camera.py

```diff
@@ -0,0 +1,270 @@
+# Generates Stage framing camera.
+# heavy inspiration taken from: 
+# https://github.com/beersandrew/assets/tree/1df93f4da686e9040093027df1e42ff9ea694866/scripts/thumbnail-generator
+
+import math
+from typing import Union
+
+from pxr import Usd, UsdGeom
+from pxr import Sdf, Gf
+
+def _stage_up(stage: Usd.Stage) -> str:
+    return UsdGeom.GetStageUpAxis(stage)
+
```

Comment:
Ensure PEP08 - two spaces surrounding top-level functions
```suggestion


def _stage_up(stage: Usd.Stage) -> str:
    return UsdGeom.GetStageUpAxis(stage)


```

Reply:
implemented in next push
added padding in top level function `_setup_opengl_widget` in playblast submodule as well.